search-domain/build.gradle

```diff
@@ -1,3 +1,46 @@
+buildscript {
```

Comment:
Most of these should be changed in the project wide `build.gradle`, not added in the subproject specific build files.